### DISCUSSION TITLE
nessma tv url !

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#117] [Show and tell]
the tunisians channel nessma tv return after a break .
the new name is " nessma ejdida " mean the new nessma 
https://www.nessma.tv/ar/live

**Comment by BellezaEmporium** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x7va0xb?embedder=https%3A%2F%2Fwww.nessma.tv%2F&referer=&dmTs=486299&is_native_app=0&dmV1st=EAC1177DE42B93A706FE6CF90CC8EAC4

**Reply by ARAB-IPTV** [reply]
the link don't play !

**Reply by BellezaEmporium** [reply]
Worked for me when I tried it last time

**Reply by Leclowndu93150** [reply]
man do you havee any m3u for that channel pls ?

**Reply by ARAB-IPTV** [reply]
> man do you havee any m3u for that channel pls ?

http://217.182.137.206/nessma.m3u8
But Not 24/7

**Reply by BellezaEmporium** [reply]
> > man do you havee any m3u for that channel pls ?
> 
> http://217.182.137.206/nessma.m3u8 But Not 24/7

yeah it doesn't seem to work properly haha.

other channels linked to this server : 
http://217.182.137.206/tunisie2.m3u8
http://217.182.137.206/elhiwar.m3u8
http://217.182.137.206/sudan_drama.m3u8

**Reply by Leclowndu93150** [reply]
ty

**Comment by Sphinxroot** [comment]
https://www.dailymotion.com/video/x7va0xb  offline for now  so not 24/7
